app/collins/models/shared/IpAddressable.scala

```diff
@@ -127,11 +127,11 @@ trait IpAddressStorage[T <: IpAddressable] extends Schema with AnormAdapter[T] w
   * For a range 0L..20L, used addresses List(17,18,19,20), the result will be None (allocate from beginning)
   */
   protected def getCurrentLowestLocalMaxAddress(calc: IpAddressCalc)(implicit scope: Option[String]): Option[Long] = inTransaction {
-    val minAddress = calc.minAddressAsLong
+    val startAddress = calc.startAddressAsLong
```

Comment:
Well, yes.  But that is the way it was before as well - if you did not specify a gateway or a start address, both would be set to `minAddress`.  I'm not sure if collins would actually hand out the min address in that scenario or not.